eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/legacyjpa/EmbeddedEventStore.java

```diff
@@ -1,640 +0,0 @@
-/*
```

Comment:
Although the window of opportunity is small, I don't think we can remove this, as that may just introduce a breaking change.

Reply:
Reverted the remove.